vendor/vendor.json

```diff
@@ -2284,6 +2311,12 @@
 			"version": "v19.03.5",
 			"versionExact": "v19.03.5"
 		},
+		{
+			"checksumSHA1": "Yl6cD918tLOXa0I/iuGiovmszQU=",
+			"path": "github.com/docker/docker/pkg/pools",
```

Comment:
Is this import correct? The field "origin" is missing and its version differs from the one we use (v19.03.5).